draft-packages/select/package.json

```diff
@@ -38,9 +38,9 @@
     "@kaizen/draft-form": "^3.6.3",
     "@kaizen/draft-tag": "^1.9.3",
     "@types/classnames": "^2.2.10",
-    "@types/react-select": "^3.0.22",
+    "@types/react-select": "^4.0.17",
     "classnames": "^2.2.6",
-    "react-select": "^3.1.0"
+    "react-select": "^4.3.1"
```

Comment:
I was a little concerned about this change, but it doesn't appear any of the breaking changes are applicable to our implementation: https://github.com/JedWatson/react-select/blob/master/packages/react-select/CHANGELOG.md#400